scalac-scoverage-plugin/src/main/scala/scoverage/Serializer.scala

```diff
@@ -22,6 +22,14 @@ object Serializer {
   }
 
   def serialize(coverage: Coverage, writer: Writer): Unit = {
+    // Used for getting the relative filepath for [stmt.location.sourcePath]
+    // instead of the canonical path. This is required to make it work with
+    // remoting or in a distributed environment.
+    def getRelativePath(filePath: String): String = {
+      val base = new File(".").getCanonicalPath + File.separator
```

Comment:
This operation is `relativize`: https://docs.oracle.com/javase/tutorial/essential/io/pathOps.html#relativize